src/transforms/route.rs

```diff
@@ -38,15 +39,17 @@ impl SyncTransform for Route {
         event: Event,
         output: &mut vector_core::transform::TransformOutputsBuf,
     ) {
+        let mut passed: usize = 0;
         for (output_name, condition) in &self.conditions {
             if condition.check(&event) {
                 output.push_named(output_name, event.clone());
             } else {
-                emit!(&RouteEventDiscarded {
```

Comment:
Should we document (i.e. in the upgrade guide) that this metric is disappearing for `route`?

Reply:
🤔 Do you know where I should do this?

Reply:
https://github.com/vectordotdev/vector/blob/master/website/content/en/highlights/2022-03-22-0-21-0-upgrade-guide.md <- assuming this is going to make it into the 0.21.0 release!

Reply:
This seems likely to make it into 0.21.0 so you can just put it in that upgrade guide. We are still waiting for a couple of issues to be wrapped up before release.